src/app/calendar/calendar.component.ts

```diff
@@ -60,37 +76,76 @@ export class CalendarComponent implements OnInit {
     let firstDay: Moment = moment(dateInMonth).startOf('month').startOf('week');
     let lastDay: Moment = moment(dateInMonth).endOf('month').endOf('week');
 
-    this.days = [];
-    for (let d: Moment = firstDay.clone(); d.isBefore(lastDay); d.add(1, 'days')) {
-      // console.log(this.getEventsOnDate(d));
-      console.log('no');
-      let calendarDay: CalendarDay = {
-        dayOfMonth: d.date(),
-        inCurrentMonth: d.isSame(dateInMonth, 'month'),
-        events: this.getEventsOnDate(d),
-      };
-
-      if (d == moment()){
-        this.today = calendarDay;
-      }
+    // this.days = [];
```

Comment:
Let's avoid commenting code, we can use git history to get it back if we really need it